system-variables.md

```diff
@@ -224,20 +240,23 @@ This variable is an alias for `last_insert_id`.
 ### init_connect
 
 - Scope: GLOBAL
+- Persists to cluster: Yes
 - Default value: ""
 - The `init_connect` feature permits a SQL statement to be automatically executed when you first connect to a TiDB server. If you have the `CONNECTION_ADMIN` or `SUPER` privileges, this `init_connect` statement will not be executed. If the `init_connect` statement results in an error, your user connection will be terminated.
 
 ### innodb_lock_wait_timeout
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `50`
-- Range: `[1, 1073741824]`
+- Range: `[1, 3600]`
```

Comment:
This is from https://github.com/pingcap/tidb/pull/33473 - it only affects master / TiDB 6.1.